samplejava/build.gradle

```diff
@@ -36,12 +34,8 @@ android {
 dependencies {
     // Add new dependencies
     implementation "io.getstream:stream-chat-android-ui-components:4.5.4"
-    implementation 'io.coil-kt:coil:1.1.1'
 
-    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'io.coil-kt:coil:1.1.1'
     implementation 'com.google.android.material:material:1.3.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    testImplementation 'junit:junit:4.+'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
```

Comment:
Let's keep all the default dependencies in here, and just add the new ones in above as required

```
// Add new dependencies
<new deps we add>

<original deps>
```

Reply:
Sorry, I finally got the idea